doc/man/tracef.3.txt

```diff
@@ -0,0 +1,118 @@
+tracef(3)
+=========
+:object-type: macro
+
+
+NAME
+----
+tracef - LTTng-UST man:printf(3)-like interface
+
+
+SYNOPSIS
+--------
+[verse]
+*#include <lttng/tracef.h>*
+
+[verse]
+#define *tracef*('fmt', ...)
+
+Link with `-llttng-ust`.
```

Comment:
`-ldl` is only required when defining `TRACEPOINT_DEFINE` or `TRACEPOINT_CREATE_PROBES` before including `<lttng/tracepoint.h>`, which is not the case with the include chain starting at `<lttng/tracef.h>`.